src/java/picard/sam/FilterSamReads.java

```diff
@@ -79,12 +83,12 @@
             "<hr />";
     private static final Log log = Log.getInstance(FilterSamReads.class);
 
-    private static enum Filter {
+    public static enum Filter {
```

Comment:
Please revert it to private if you can (looks like you can).  Also, could you remove the static keyword while you're at it?  It's redundant.